### ISSUE TITLE
Adding a Carthage dependency from a subdirectory of another git repo

### ISSUE THREAD

**Issue by manojmahapatra** [Carthage/Carthage#2814]
* carthage install method: [ ] .pkg, [x] homebrew, [ ] source
* `which carthage`: /usr/local/bin/carthage
* `carthage version`: 0.33.0
* `xcodebuild -version`: Xcode_10
* Are you using `--no-build`? N/A
* Are you using `--no-use-binaries`? N/A
* Are you using `--use-submodules`? N/A
* Are you using `--cache-builds`? N/A
* Are you using `--new-resolver`? N/A

**Cartfile**
```
git "https://foo/bar/subDirectory" "master"
```

**Carthage Output**
```
failed with exit code 128
```

**Actual outcome**
Carthage did not bootstrap/update dependency from a subdirectory of another git repo. 

**Expected outcome**
not sure (i guess this should work)

This could be a duplicate issue, while I searched and did not find anything close. hence I opened the issue. 
Basically, I want to make/add a Carthage (framework) dependency from a subdirectory of another git repo. This subdirectory (has a subproject e.g. `bar.xcodeproj`) is part of a workspace (e.g. `Foo.xcworkspace`) which has few other `xcodeproj` too.

So my question is, how can I able to add a subproject from a subdirectory of another git repo. 

Thanks

**Comment by blender** [comment]
Sorry but this is not supported.

**Comment by manojmahapatra** [comment]
thanks @tmspzz, closing this issue.
